lib/ansible/modules/monitoring/zabbix/zabbix_screen.py

```diff
@@ -414,8 +440,14 @@ def main():
             graphs_in_row = zabbix_screen['graphs_in_row']
             graph_width = zabbix_screen['graph_width']
             graph_height = zabbix_screen['graph_height']
-            host_group_id = screen.get_host_group_id(host_group)
-            hosts = screen.get_host_ids_by_group_id(host_group_id, sort)
+            try:
+                host_group = zabbix_screen['host_group']
+                host_groups = host_group if isinstance(host_group, list) else [host_group]
+            except KeyError:
+                host_groups = zabbix_screen['host_groups']
+                host_groups = host_groups if isinstance(host_groups, list) else [host_groups]
```

Comment:
There is no benefit to such handling of "hidden" module parameter. `AnsibleModule` should be fairly capable of doing this for us. What I mean is this:

```python
    module = AnsibleModule(
        argument_spec=dict(
            host_group=dict(type='list', aliases=['host_groups']),
        ),
    )

    module.exit_json(ok=True, host_group=module.params['host_group'])
```

Will accept parameters in both formats and convert them to list:
```yaml
    - testmod:
        host_group: abc

    - testmod:
        host_group:
          - abc
          - def
```

Reply:
@D3DeFi Thanks for this, removed the useless try/catch and replaced with changing host_group str->list + added the alias.